consensus/consensus-types/src/block.rs

```diff
@@ -170,7 +170,7 @@ where
             ),
         );
 
-        let block_data = BlockData::new_genesis(ledger_info.timestamp_usecs(), genesis_quorum_cert);
+        let block_data = BlockData::new_genesis(0, genesis_quorum_cert);
```

Comment:
Would prefer this either be system-wide const or dependent on some other form of determinism (such as it currently is)